qiskit/chemistry/qubit_transformations/fermionic_transformation.py

```diff
@@ -0,0 +1,518 @@
+                and be the correct symmetry sector needed for the ground state. Setting 'auto' will
+                use an automatic computation of the correct sector. If from other experiments, with
+                the z2symmetry logic, the sector is known, then the tapering values of that sector
+                can be provided (a list of int of values -1, and 1). The default is None
+                meaning no symmetry reduction is done. Note that dipole and other operators
+                such as spin, num particles etc are also symmetry reduced according to the
+                symmetries found in the main operator if this operator commutes with the main
+                operator symmetry. If it does not then the operator will be discarded since no
+                meaningful measurement can take place.
+        Raises:
+            QiskitChemistryError: Invalid symmetry reduction
+        """
+        transformation = transformation.value
+        orbital_reduction = orbital_reduction if orbital_reduction is not None else []
+        super().__init__()
+        self._transformation = transformation
+        self._qubit_mapping = qubit_mapping.value
+        self._two_qubit_reduction = two_qubit_reduction
+        self._freeze_core = freeze_core
+        self._orbital_reduction = orbital_reduction
+        if z2symmetry_reduction is not None:
+            if isinstance(z2symmetry_reduction, str):
+                if z2symmetry_reduction != 'auto':
+                    raise QiskitChemistryError('Invalid z2symmetry_reduction value')
+        self._z2symmetry_reduction = z2symmetry_reduction
+
+        # Store values that are computed by the classical logic in order
+        # that later they may be combined with the quantum result
+        self._hf_energy = None
+        self._nuclear_repulsion_energy = None
+        self._nuclear_dipole_moment = None
+        self._reverse_dipole_sign = None
+        # The following shifts are from freezing orbitals under orbital reduction
+        self._energy_shift = 0.0
+        self._x_dipole_shift = 0.0
+        self._y_dipole_shift = 0.0
+        self._z_dipole_shift = 0.0
+        # The following shifts are from particle_hole transformation
+        self._ph_energy_shift = 0.0
+        self._ph_x_dipole_shift = 0.0
+        self._ph_y_dipole_shift = 0.0
+        self._ph_z_dipole_shift = 0.0
+
+        self._molecule_info: Dict[str, Any] = {}
+
+    def transform(self, driver: BaseDriver) -> Tuple[WeightedPauliOperator,
+                                                     List[WeightedPauliOperator]]:
+        """
+        Transformation to qubit operator from the driver
+
+        Args:
+            driver: Base Driver
+
+        Returns:
+            qubit operator, auxiliary operators
+        """
+        q_molecule = driver.run()
+        ops, aux_ops = self._do_transform(q_molecule)  # _do_transform(q_molecule)
```

Comment:
I assume this comment is some editing leftover - seems like it should be deleted as it has no value